### ISSUE TITLE
Tactical recommendations needs additional info on parties

### ISSUE THREAD

**Issue by baob** [swapmyvote/swapmyvote#230]
Follow on to #161

Basics of #161 are complete, however we haven't yet addressed

* [x] translating recommendation text into a party name, where possible
* [x] using the party name to colour the recommended party background
* [ ] standardising on party names all caps vs mixed case, BXP versus Brexit, LD versus LibDems etc

There may be more.  @tdg ? @aspiers ?

#227 needs to be merged before this issue can be completed, but it can be started at any time.

**Comment by aspiers** [comment]
Sounds like M2 or even M3 to me but @tdg should have final say.

**Comment by tdg** [comment]
It's more useful the longer we have it this election, so early M2 probably.
Colour will make a lot of difference

**Comment by tdg** [comment]
If we can have some info ideally embedded via a tooltip it might also help with impact/ understanding #220

**Comment by baob** [comment]
Work was started a day or two ago ... but still subject to change.

https://github.com/swapmyvote/swapmyvote/compare/master...baob:issue-161-parties-for-recommendations?expand=1

**Comment by aspiers** [comment]
This is already awesome and the final cosmetics are way lower priority than other stuff - moving to M4.

**Comment by baob** [comment]
When the PR for issue #773 is merged, then this can be closed (final checkbox done)

**Comment by baob** [comment]
Closed by merge of #791
